### PR TITLE
fix(wallet-mobile): qr theming

### PR DESCRIPTION
**Pull request by banklesss**
## Description / Change(s) / Related issue(s)

Adding correct qr colors

##### Ticket

YOMO-1867

> [!NOTE]
> Please create the ticket if missing it.

### CODE REVIEW

File: apps/wallet-mobile/src/features/Exchange/useCases/CreateExchangeOrderScreen/CreateExchangeOrderScreen.tsx [hunk 1]
```diff
@@ -102,6 +102,8 @@ export const CreateExchangeOrderScreen = () => {
         openModal('', <ErrorScreen />, undefined, undefined, true)
       },
       onSuccess: (referralLink) => {
+        closeModal()
```
Comment (banklesss) [inline apps/wallet-mobile/src/features/Exchange/useCases/CreateExchangeOrderScreen/CreateExchangeOrderScreen.tsx:105]: missed this one 👀

**Review by stackchain — APPROVED** [review]

File: apps/wallet-mobile/src/features/Exchange/useCases/CreateExchangeOrderScreen/CreateExchangeOrderScreen.tsx (hunk 1 above)
Comment (stackchain) [reply apps/wallet-mobile/src/features/Exchange/useCases/CreateExchangeOrderScreen/CreateExchangeOrderScreen.tsx:105]: navigation for rescuing :D

**Comment by banklesss** [comment]
fixing one thing

**Comment by banklesss** [comment]
> fixing one thing

done

**Comment by stackchain** [comment]
> fixing one thing

all backgrounds need to be white

**Comment by banklesss** [comment]
yeah!!! thats the fix 👀

File: apps/wallet-mobile/src/features/Transactions/useCases/TxDetails/AddressModal/AddressModal.tsx [hunk 2]
```diff
@@ -142,5 +142,13 @@ const useStyles = () => {
       color: color.gray_c900,
     },
   })
-  return {styles}
+
+  const colors = {
+    white: color.gray_cmin,
+    black: color.gray_cmax,
```
Comment (banklesss) [inline apps/wallet-mobile/src/features/Transactions/useCases/TxDetails/AddressModal/AddressModal.tsx]:
```suggestion
    white: color.white_static,
    black: color.black_static,
```

**Pull request merged**